src/docbuilder/mod.rs

```diff
@@ -1,12 +1,10 @@
 mod crates;
 mod limits;
-mod metadata;
 pub(crate) mod options;
 mod queue;
 mod rustwide_builder;
 
 pub(crate) use self::limits::Limits;
-pub(self) use self::metadata::Metadata;
```

Comment:
I'd remove the re-exporting of a library if at all possible

Reply:
It is removed? I don't understand the comment.

Reply:
The GH app was lying to me, never mind